src/pypgstac/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "pgstacrs"
```

```suggestion
name = "pypgstac"
```

We already have a **pgstac**: https://crates.io/crates/pgstac. The `rs` is redundant, since this is a Rust package definition. IMO `pypgstac` would be the best Rust package name.